templates/batch-llm/README.ipynb

```diff
@@ -101,7 +153,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "NOTE: This notebook uses local text data defined above, but it can easily be expanded to read from input data stored in cloud storage. To read text files from cloud storage (e.g., AWS S3), you can modify the dataset to be:\n",
+    "_Note_: This notebook uses local text data defined above, but it can easily be expanded to read from input data stored in cloud storage. To read text files from cloud storage (e.g., AWS S3), you can instead define the dataset as:\n",
```

Comment:
Each template should have a scaling out section, that provides a larger-scale run example (could still use dummy data).

Reply:
Created a scaling up section using the additional `.txt` file on S3 (5800 rows).